morphir/runtime/src/org/finos/morphir/runtime/quick/EvaluatorQuick.scala

```diff
@@ -155,7 +156,12 @@ object EvaluatorQuick {
         Data.Int32(value.toInt)
       case (Concept.Int64, RTValue.Primitive.Int(value)) =>
         Data.Int64(value.toLong)
-
+      case (Concept.Order, RTValue.ConstructorResult(FQString("Morphir.SDK:Basics:gT"), List())) =>
```

Comment:
They need the weird capitalization because of how FQName.toString assumes local names to be camelCased, and because of how the FQString extractor works.

Basically, if you do `fqn"Morphir.SDK:Basics:LT"` that becomes an FQN whose localName is `["l", "t"]`, and then if you call toString on that you get back `fqn"Morphir.SDK:Basics:lT"`. Other code doesn't show it because it doesn't use the default `toString` structure, but context-specific capitalization schemes. This code uses the `FQString` extractor, which is there to make pattern matching against FQNames easy (when you can't just construct them as part of an extractor, given their non-trivial structure). 

I didn't love writing it this way, and considered changing the extractor code to something else, but I don't think the added complexity would be worth it when it only impacts this one case.